### DISCUSSION TITLE
개인 안심 번호 유명 무실 기사 공유

### DISCUSSION THREAD

**Discussion by designe** [euphony-io/check-in-listener#35] [General]
개인 안심 번호가 만들어졌지만 사실상 거의 쓰이지 않고 있다는 기사입니다 :)
우리가 이걸 살려주는 역할이 되면 좋겠네요 :)

https://www.koit.co.kr/news/articleView.html?idxno=86136

■수기명부에 휴대폰 번호만 가득

다중이용시설에 들어갈 때 QR코드 기기가 비치돼 있으면 그것을 이용하면 된다.

하지만 QR코드 기기가 없는 경우에는 수기명부를 작성해야 한다.

개인안심번호를 시행한 2월부터 지금까지 다중시설을 자주 이용했다.

그때마다 그곳에 놓여있는 수기 명부를 여러 번 확인해 봤지만 개인안심번호를 사용하는 사람은 거의 없었다.

시행 4개월이 지나고 있는 지금 별반 달라진 게 없었다.

최근 점심을 먹기 위해 방문한 종로3가역 인근 식당에 들어가 수기명부를 작성하려고 보니 휴대폰번호가 빼곡히 적혀 있고 개인안심번호가 적혀 있는 건 찾을 수가 없었다.

식당 주인에게 "개인안심번호를 적는 사람은 거의 없네요" 물었다.

주인은 "그거 쓰는 사람도 별로 없고 나도 어느 손님이 알려줘서 개인안심번호라는 걸 알았다"고 말했다.

퇴근길에 사람들이 많이 몰리는 사당역 근처 식당에 들러서 살펴보니 그곳에도 개인안심번호를 적은 사람은 없었다.

식당에 들어와서 수기명부에 휴대폰 번호를 적으려는 손님에게 전화번호 말고 개인안심번호를 적으면 번호 노출이 안돼서 좋다고 설명했다.

그는 "개인정보 노출이 안 되서 참 좋은데 왜 지금까지 이런 게 있는지 몰랐을까요 고맙습니다"고 말했다.

개인정보 노출이 안 되는 개인안심번호의 인지도가 낮아 답답한 마음에 여러 지인들에게 개인안심번호를 아느냐고 묻자 "그게 뭔데 모른다"는 답변만 들었다.

출처 : 정보통신신문(http://www.koit.co.kr)

**Comment by Jsueeee** [comment]
@euphony-io/eu-us 
다른 분들은 개인안심번호 적으시는 편이신가요?
저도 생각해보면 앱 켜서 큐알코드 찍는 편인 것 같습니다 ㅠㅠ!
종이에 안심번호 보고 적는 것보다 이게 편해서 그런 것 같아요.
저희 앱이 해결방안이 됐으면 좋겠네요😄

**Comment by dlwlstks96** [comment]
이전에는 개인안심번호란게 존재한단 사실만 얼핏 알았고
그것을 수기명부에 대신 작성해도 되는지는 프로젝트 시작하고 나서 알게 되었습니다!!
저희 프로젝트가 성공적으로 마무리되고 널리 알려졌으면 좋겠네요ㅎㅎ
